src/date.ts

```diff
@@ -0,0 +1,175 @@
+import 'globalize';
+import 'globalize/dist/globalize/date';
```

Comment:
As far as I can tell, no. In an AMD environment, the loader `paths` can be easily updated to allow this. In a Node environment, however, `require('globalize')` loads the entire Globalize.js package, so that `globalize/date`, etc. are automatically included. As such, `globalize/date` is not a module ID that Node can find. Globalize.js itself can be used in either CommonJS or AMD environments, but is not structured in a way that consuming packages can themselves be isomorphic without a bit of ugliness.